app/views/spam2/_nodes.html.erb

```diff
@@ -1,45 +1,47 @@
 <script>
 $(document).ready(function () {
-	var table = table_main("#node_table");
-	$("#all").click(function () { //select all button
+  var table = table_main("#node_table");
+  $("#all").click(function () { //select all button
     select_all();
   });
+    
```

Comment:
@mr-funtor Can you remove this empty line? Everything else looks good. Thanks!!

Reply:
@TildaDares I have removed the line. Thank you